app/models/application_record.rb

```diff
@@ -1,3 +1,18 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
+
+  class << self
```

Comment:
Why is this needed?

Reply:
prefer to delcare this insted of `def self.method_name`